### PR TITLE
pythonPackages.ronin: init at 1.1.1

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/ronin/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    inherit version pname;
+    hash = "sha256-5gZ8S0NR4JzKBIdi/xYtVmFg9ObbCSkT7sz+OKWnK/U=";
+  };
+
+  propagatedBuildInputs = [
+    blessings
+    colorama
+    glob2
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ronin/default.nix:23]: Missing a pythonImportsCheck and checkPhase.

Comment (AndersonTorres) [reply pkgs/development/python-modules/ronin/default.nix:23]: There is no testsuite on this package. I will add a typical pythonImportsCheck.

File: pkgs/development/python-modules/ronin/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  pythonImportsCheck = [
+    "blessings"
+    "colorama"
+    "glob2"
```
Comment (drewrisinger) [inline pkgs/development/python-modules/ronin/default.nix:27]:
```suggestion
```
Usually (for me at least), I just check that the package I'm building imports, not the dependencies.

File: pkgs/development/python-modules/ronin/default.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "ronin"
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/tliron/ronin/";
+    description = "A straightforward but powerful build system based on Ninja and Python";
+    license = licenses.asl20;
```
Comment (drewrisinger) [inline pkgs/development/python-modules/ronin/default.nix:34]: Does it have a changelog?

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [ ] importscheck might need changed?
* [x] Commits LGTM
* [x] Builds via ``nix-review`` on x86-64-linux:
```
2 packages built:
python38Packages.ronin python39Packages.ronin
```

File: pkgs/development/python-modules/ronin/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/ronin/default.nix:27]: That is the correct usage you described.

File: pkgs/development/python-modules/ronin/default.nix (hunk 3 above)
Comment (AndersonTorres) [reply pkgs/development/python-modules/ronin/default.nix:34]: Nope.

**Pull request merged**